armi/operators/tests/test_operators.py

```diff
@@ -207,15 +207,29 @@ def test_getActiveInterfaces(self):
         self.assertIn("history", interfaceNames)
         self.assertNotIn("xsGroups", interfaceNames)
 
-        # Test EOL
-        interfaces = self.o.getActiveInterfaces("EOL")
-        self.assertEqual(interfaces[-1].name, "main")
-
         # Test Coupled
         interfaces = self.o.getActiveInterfaces("Coupled")
         for test, ref in zip(interfaces, self.activeInterfaces):
             self.assertEqual(test.name, ref.name)
 
+        # Test EOL
+        interfaces = self.o.getActiveInterfaces("EOL")
+        self.assertEqual(interfaces[-1].name, "main")
+        print([i.name for i in interfaces])
```

Comment:
quick scan. I think this can probably be deleted
```suggestion
```

Reply:
D'oh!